jsons/manifesto/it.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title":"Manifesto per l'Artigianato del Software",
+  "raising_the_bar":"Alzare l'asticella.",
+  "introduction":"Come aspiranti Artigiani del Software, stiamo alzando l'asticella dello sviluppo professionale del software praticandolo e aiutando gli altri a imparare il mestiere.  Attraverso questo lavoro siamo arrivati a dare valore a:",
+  "not_only":"Non solo ",
+  "but_also":"ma anche ",
+  "working_software":"software funzionante,",
+  "well_crafted_software":"software ben fatto",
+  "responding_to_change":"rispondere al cambiamento,",
+  "steadily_adding_value":"valore aggiunto costante",
+  "individuals_and_interactions":"individui e interazioni,",
+  "a_community_of_professionals":"una comunità di professionisti",
+  "customer_collaboration":"collaborazione con i clienti,",
+  "productive_partnerships":"partenariati produttivi",
```

Comment:
This is tough. I understand why you are using "partenariati produttivi" but it's not used in common language.

My suggestion is "Non solo relazione (o interazione) con il cliente, ma collaborazione produttiva", alternatively we can use "partnership" also in italian, since is also an italian word. In this case we can say "Non solo collaborazione con il cliente, ma anche partnership produttiva".

Reply:
Agreed for "Non solo collaborazione con il cliente, ma anche partnership produttiva".

Reply:
I agree with the approach, I also like "Non solo collaborazione con il cliente, ma anche partnership produttiva" translation, the first I proposed diverges a bit from the english one.